.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @paritytech/opstooling
```

Comment:
> Unknown owner on line 1: make sure the team @paritytech/opstooling exists, is publicly visible, and has write access to the repository

Reply:
I think that it is failing because it was set in the other organization. Should work once merged in ours

Reply:
i guess the opstooling should have admin access in this repo and it should be ok